pkg/util/provider/machinecontroller/machine_util.go

```diff
@@ -914,13 +914,15 @@ func (c *controller) drainNode(deleteMachineRequest *driver.DeleteMachineRequest
 		state              v1alpha1.MachineState
 
 		// Initialization
-		machine                 = deleteMachineRequest.Machine
-		maxEvictRetries         = int32(math.Min(float64(*c.getEffectiveMaxEvictRetries(machine)), c.getEffectiveDrainTimeout(machine).Seconds()/drain.PodEvictionRetryInterval.Seconds()))
-		pvDetachTimeOut         = c.safetyOptions.PvDetachTimeout.Duration
-		timeOutDuration         = c.getEffectiveDrainTimeout(deleteMachineRequest.Machine).Duration
-		forceDeleteLabelPresent = machine.Labels["force-deletion"] == "True"
-		nodeName                = machine.Labels["node"]
-		nodeNotReadyDuration    = 5 * time.Minute
+		machine                     = deleteMachineRequest.Machine
+		maxEvictRetries             = int32(math.Min(float64(*c.getEffectiveMaxEvictRetries(machine)), c.getEffectiveDrainTimeout(machine).Seconds()/drain.PodEvictionRetryInterval.Seconds()))
+		pvDetachTimeOut             = c.safetyOptions.PvDetachTimeout.Duration
+		timeOutDuration             = c.getEffectiveDrainTimeout(deleteMachineRequest.Machine).Duration
+		forceDeleteLabelPresent     = machine.Labels["force-deletion"] == "True"
+		nodeName                    = machine.Labels["node"]
+		nodeNotReadyDuration        = 5 * time.Minute
+		readOnlyFileSystemCondition v1.NodeCondition
+		nodeReadyCondition          v1.NodeCondition
```

Comment:
```suggestion
		readOnlyFileSystemCondition, nodeReadyCondition          v1.NodeCondition
```
You can move this to the declaration section just above this at line 915